### PR TITLE
Update delegates-overview.md

### PR DESCRIPTION
**Pull request by pratmit**
Updated the video and delegate sizing screenshot. Can you help me updating the content in a tabular format?

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 1]
```diff
@@ -75,7 +75,21 @@ For advanced installation topics, see the following:
 
 One Delegate size does not fit all use cases, so Harness let's you pick from several options:
 
-![](./static/delegates-overview-01.png)
+Replicas
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:78]:
```suggestion
| Replicas | Required memory / CPU | Maximum parallel deployments and builds across replicas |
| :--: |  :--: | :--: |
| 1 | 2 GB / 0.5 CPU | 25 |
| 2 | 4 GB / 1 CPU | 50 |
| 4 | 8 GB / 2 CPU | 100 |
| 8 | 16 GB / 4 CPU | 200 |
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 2]
```diff
@@ -75,7 +75,21 @@ For advanced installation topics, see the following:
...
+Total Memory / Total CPU needed on the Kubernetes Cluster
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:79]:
```suggestion

```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 3]
```diff
@@ -75,7 +75,21 @@ For advanced installation topics, see the following:
 
 One Delegate size does not fit all use cases, so Harness let's you pick from several options:
 
-![](./static/delegates-overview-01.png)
+Replicas
+Total Memory / Total CPU needed on the Kubernetes Cluster 
+Number of parallel deployments/builds that can be run across all replicas
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:80]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 4]
```diff
@@ -75,7 +75,21 @@ For advanced installation topics, see the following:
...
+1
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:81]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 5]
```diff
@@ -75,7 +75,24 @@ For advanced installation topics, see the following:
...
 
-![](./static/delegates-overview-01.png)
+| Replicas | Required memory / CPU | Maximum parallel deployments and builds across replicas |
+| :--: |  :--: | :--: |
+| 1 | 2 GB / 0.5 CPU | 25 |
+| 2 | 4 GB / 1 CPU | 50 |
+| 4 | 8 GB / 2 CPU | 100 |
+| 8 | 16 GB / 4 CPU | 200 |
+Number of parallel deployments/builds that can be run across all replicas
+2 GB / 0.5 CPU
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:85]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 6]
```diff
@@ -75,7 +75,24 @@ For advanced installation topics, see the following:
...
+25
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:86]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 7]
```diff
@@ -75,7 +75,24 @@ For advanced installation topics, see the following:
...
+2
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:87]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 8]
```diff
@@ -75,7 +75,24 @@ For advanced installation topics, see the following:
...
+4 GB / 1 CPU
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:88]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 9]
```diff
@@ -75,7 +75,24 @@ For advanced installation topics, see the following:
...
+50
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:89]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 10]
```diff
@@ -75,7 +75,24 @@ For advanced installation topics, see the following:
...
+4
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:90]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 11]
```diff
@@ -75,7 +75,24 @@ For advanced installation topics, see the following:
...
+8 GB / 2 CPU
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:91]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 12]
```diff
@@ -75,7 +75,24 @@ For advanced installation topics, see the following:
...
+100
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:92]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 13]
```diff
@@ -75,7 +75,24 @@ For advanced installation topics, see the following:
...
+8
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:93]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 14]
```diff
@@ -75,7 +75,24 @@ For advanced installation topics, see the following:
...
+16 GB / 4 CPU
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:94]:
```suggestion
```

File: docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md [hunk 15]
```diff
@@ -75,7 +75,24 @@ For advanced installation topics, see the following:
...
+200
```
Comment (kat-enos) [inline docs/platform/2_Delegates/get-started-with-delegates/delegates-overview.md:95]:
```suggestion
```

**Review by kat-enos — DISMISSED** [review]

**Review by pratmit — COMMENTED** [review]
LGTM

**Comment by kat-enos** [comment]
Thank you @pratmit . Merging.

**Review by kat-enos — APPROVED** [review]

**Pull request merged**